workflow-steps/cache/hashing-utils.ts

```diff
@@ -2,6 +2,9 @@ const fs = require('fs');
 const crypto = require('crypto');
 import { glob } from 'glob';
```

Comment:
not your change, but this being an `import` while everything else is a `require` caught my eye. you can change it leave it. doesn't matter, but needed to call it out for my sanity